File: php/arrays/adding_elements.php
#!/usr/bin/php8.0
<?php

$array = [2, 4, 6];

print("Initial state:\n\n");
print_r($array); print("\n");

$array[] = 8;

print("After add by \$array[] = 8:\n\n");
print_r($array); print("\n");

array_push($array, 10);

print("After add by array_push(\$array[], 10):\n\n");
print_r($array); print("\n");

array_push($array, 12, 14, 16);

print("After add by array_push(\$array[], 12, 14, 16):\n\n");
print_r($array); print("\n");

array_unshift($array, 0);

print("After add by array_unshift(\$array, 0):\n\n");
print_r($array); print("\n");

array_unshift($array, -6, -4, -2);

print("After add by array_unshift(\$array, -6, -4, -2):\n\n");
print_r($array); print("\n");

File: ruby/arrays/arrays/operations_on_elements/adding_elements.rb
#!/usr/bin/ruby2.7

array = [2, 4, 6]

puts("Initial state:\n\n")
puts(array.inspect, "\n")

# << operator

array << 8

puts("After add by array << 8:\n\n")
puts(array.inspect, "\n")

# append

array.append(10)

puts("After add by array.append(10):\n\n")
puts(array.inspect, "\n")

array.append(12, 14, 16)

puts("After add by array.append(12, 14, 16):\n\n")
puts(array.inspect, "\n")

# push

array.push(18)

puts("After add by array.push(18):\n\n")
puts(array.inspect, "\n")

array.push(20, 22, 24)

puts("After add by array.push(20, 22, 24):\n\n")
puts(array.inspect, "\n")

# prepend

array.prepend(0)

puts("After add by array.prepend(0):\n\n")
puts(array.inspect, "\n")

array.prepend(-6, -4, -2)

puts("After add by array.prepend(-6, -4, -2):\n\n")
puts(array.inspect, "\n")

# unshift

array.unshift(-8)

puts("After add by array.unshift(-8):\n\n")
puts(array.inspect, "\n")

array.unshift(-14, -12, -10)

puts("After add by array.unshift(-14, -12, -10):\n\n")
puts(array.inspect, "\n")

# insert

array.insert(1, 26)

puts("After add by array.insert(1, 26):\n\n")
puts(array.inspect, "\n")

array.insert(21, 28)

puts("After add by array.insert(21, 28):\n\n")
puts(array.inspect, "\n")

array.insert(24, 30)

puts("After add by array.insert(24, 30):\n\n")
puts(array.inspect, "\n")

array.insert(0, 32)

puts("After add by array.insert(0, 32):\n\n")
puts(array.inspect, "\n")

array.insert(-1, 34)

puts("After add by array.insert(-1, 34):\n\n")
puts(array.inspect, "\n")
